ui/css/logviewer.css

```diff
@@ -77,10 +77,29 @@ body {
     white-space: pre;
 }
 
+.lv-line-num {
+    white-space: pre;
+    float:left;
+    align-items: center;
+    border-right-style: solid;
+    border-right-color:  #F8EEC7;
```

Comment:
Let's try a mid-tone grey `border-right-color: #e6e6e6` here, rather than the selected yellow line color. Maybe these three lines would shorthand ok also?
`border-right: solid thin #e6e6e6;`

And let's add a text `color: rgba(0, 0, 0, 0.3);` property at the bottom of the class block. So the net of that change should look like this below with a default load (no lines selected), or other non-selected lines in a page with selected #LN-LY lines:

![defaultlinenumbers](https://cloud.githubusercontent.com/assets/3660661/9864952/52981bd6-5b1d-11e5-90a7-a61b0a627f6a.jpg)